docs/cadc/cadc.rst

```diff
@@ -0,0 +1,564 @@
+    https://www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/data/pub/CFHT/2376828o.fits.fz?RUNID=bxpv43misqekd16f
+    https://www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/data/pub/CFHT/2376828o_preview_1024.jpg?RUNID=bxpv43misqekd16f
+    https://www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/data/pub/CFHT/2376828o_preview_256.jpg?RUNID=bxpv43misqekd16f
+
+
+CADC data can also be queried on the target name. Note that the name is not
+resolved. Instead it is matched against the target name in the CADC metadata.
+
+.. code-block:: python
+
+  >>> from astroquery.cadc import Cadc
+  >>>
+  >>> cadc = Cadc()
+  >>> result = cadc.query_name('M31)
+  >>> print(len(result))
+
+    2000
+
+  >>> result = cadc.query_name('Nr3491_1')
+  >>> print(result)
+
+    caomObservationURI sequenceNumber proposal_keywords target_standard ... energy_transition_transition          time_bounds [2]          polarization_states      lastModified_2
+                                                                        ...                                              d
+    ------------------ -------------- ----------------- --------------- ... ---------------------------- --------------------------------- ------------------- -----------------------
+     caom:CFHT/2376828        2376828                                 0 ...                              58546.328009 .. 58546.32960815509                     2019-03-04T08:14:46.470
+
+
+Note that the examples above are for accessing data anonymously. Users with
+access to proprietary data can call ```login``` on the ```cadc``` object
+before querying or accessing the data.
+
+CADC metadata is available through a TAP service. While the above interfaces
+offer a quick and simple access to the data, the TAP interface presented in
+the next sections allows for more complex queries.
+
+=============================
+Query CADC metadata using TAP
+=============================
+
+Cadc TAP access is based on a TAP+ REST service. TAP+ is an extension of
+Table Access Protocol (TAP: http://www.ivoa.net/documents/TAP/) specified by the
+International Virtual Observatory Alliance (IVOA: http://www.ivoa.net).
+
+The TAP query language is Astronomical Data Query Language
+(ADQL: http://www.ivoa.net/documents/ADQL/2.0), which is similar
+to Structured Query Language (SQL), widely used to query databases.
+
+TAP provides two operation modes: Synchronous and Asynchronous:
+
+* Synchronous: the response to the request will be generated as soon as the
+  request received by the server.
+  (Do not use this method for queries that generate a big amount of results.)
+* Asynchronous: the server will start a job that will execute the request.
+  The first response to the request is the required information (a link)
+  to obtain the job status.
+  Once the job is finished, the results can be retrieved.
+
+The functions can be run as an authenticated user, the list_async_jobs()
```

Comment:
use backticks around function names (double backticks for code that doesn't have a doc to link to, single-backticks to link to the function's docs, e.g., `~cadc.list_async_jobs()` or whatever the full path is)

Reply:
Done